JudoPayDotNetIntegrationTests/IntegrationTestsBase.cs

```diff
@@ -15,19 +15,20 @@ namespace JudoPayDotNetIntegrationTests
     public abstract class IntegrationTestsBase
     {
         protected JudoPayApi JudoPayApiIridium;
-        protected JudoPayApi JudoPayApiCyberSource;
-        protected JudoPayApi JudoPayApiSafeCharge;
+        protected JudoPayApi JudoPayApiBase;
         protected JudoPayApi JudoPayApiElevated;
+        protected JudoPayApi JudoPayApiThreeDSecure2;
         protected readonly Configuration Configuration = new Configuration();
 
         protected IntegrationTestsBase() 
         {
             ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
-            JudoPayApiIridium = JudoPaymentsFactory.Create(Configuration.JudoEnvironment, Configuration.Token, Configuration.Secret);
-            JudoPayApiCyberSource = JudoPaymentsFactory.Create(Configuration.JudoEnvironment, Configuration.Cybersource_Token, Configuration.Cybersource_Secret);
-            JudoPayApiSafeCharge = JudoPaymentsFactory.Create(Configuration.JudoEnvironment, Configuration.SafeCharge_Token, Configuration.SafeCharge_Secret);
-            JudoPayApiElevated = JudoPaymentsFactory.Create(Configuration.JudoEnvironment, Configuration.ElevatedPrivilegesToken, Configuration.ElevatedPrivilegesSecret);
-
+            JudoPayApiBase = JudoPayApiIridium = JudoPaymentsFactory.Create(Configuration.JudoEnvironment,
```

Comment:
`JudoPayApiIridium` ?? keeping the old `AI` name eheheh.

also this line does it mean both `JudoPayApiBase` and `JudoPayApiIridium` will be assigned the result of `JudoPaymentsFactory.Create` ? don't remember if this is allowed in java but scala definitely doesn't have this.

Reply:
I want to move away from JudopayApiIridium as the credentials should not be gateway specific, but that class is referred to in many of the other integration tests classes.   To minimise the change in this PR I've introduced the new JudopayApiBase (and the syntax above sets the same factory result to both) which I've switched to use in ThreeDAuthorizationTests.cs.    As other PRs are raised on different models in this SDK I'll switch the associated tests from JudopayApiIridium to JudopayApiBase too